maps/Sudan/Sudan_conflict_points

```diff
@@ -0,0 +1,85 @@
+// Author: Iman Smith
+// Description:
+//  Script for generating conflict event mask for Sudan from 2020-2024
```

Comment:
Could you add an additional comment line with the link to latest version of this GEE script?
The link can be obtained by clicking "Get Link" button in GEE middle top.